packages/react/src/components/Slider/Slider.test.tsx

```diff
@@ -259,84 +250,74 @@ describe('Slider', () => {
       range = sliderRange;
     };
 
-    wrapper = mount(<Slider onChange={onChange} ranged defaultValue={5} />);
+    const { container } = render(<Slider onChange={onChange} ranged defaultValue={5} />);
 
-    const sliderLine = wrapper.find('.ms-Slider-line');
-    const sliderThumb = wrapper.find('.ms-Slider-slideBox');
+    const sliderLine = container.getElementsByClassName('ms-Slider-line')[0];
+    const sliderThumb = container.getElementsByClassName('ms-Slider-slideBox')[0];
 
-    sliderLine.getDOMNode().getBoundingClientRect = () =>
+    sliderLine.getBoundingClientRect = () =>
       ({ left: 0, top: 0, right: 100, bottom: 40, width: 100, height: 40 } as DOMRect);
 
-    sliderThumb.simulate('mousedown', { type: 'mousedown', clientX: 10, clientY: 0 });
-
-    sliderThumb.simulate('mousedown', { type: 'mousedown', clientX: 20, clientY: 0 });
+    fireEvent.mouseDown(sliderThumb, { clientX: 10, clientY: 0 });
+    fireEvent.mouseDown(sliderThumb, { clientX: 20, clientY: 0 });
 
     expect(range).toEqual([2, 5]);
   });
 
-  it('has type=button on all buttons', () => {
```

Comment:
Why was this removed?

Reply:
This test didn't do anything since the Slider component doesn't have any buttons

Reply:
Ha, wow... 😆